client/components/resource-details/SubmitReview.tsx

```diff
@@ -0,0 +1,403 @@
+
+const SubmitButton = styled.button`
+  font-size: 18px;
+  background-color: ${colours.primary1};
+  color: ${colours.white};
+  font-family: ${fontInter};
+  font-weight: 600;
+  padding: 5px;
+  border-radius: 5px;
+  border: 3px solid ${colours.primary1};
+  cursor: pointer;
+
+  &:hover {
+    background-color: ${colours.white};
+    color: ${colours.primary1};
+  }
+`
+
+const SubmitReviewBanner = styled.div`
+  @media ${largerThan(width.tablet)} {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  margin-bottom: 15px;
+`
+
+const ModalStyle = {
+  content: {
+    position: 'relative',
+    width: '50%',
+    minWidth: 'min(100%, ' + width.mobile + ')',
+    maxHeight: '90vh',
+    overflowX: 'auto',
+    overflowY: 'auto',
+    height: 'max(auto, 80%)',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    marginTop: '5vh',
+    textAlign: 'center',
+    inset: '0',
+  },
+  overlay: {
+    zIndex: '9',
+  },
+}
+
+const ModalHeaderRow = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 10px;
+  margin-bottom: 20px;
+`
+
+const ModalTitle = styled.h2`
+  margin: 0;
+`
+
+interface Ratings {
+  [ratingName: string]: number
+}
+
+interface AboutYouData {
+  faculty: Faculty
+  status: Status
+}
+
+const RatingLabel = styled.p`
+  margin-top: 10px;
+  margin-bottom: 10px;
+`
+
+const ModalRating = styled(
+  ({
+    className,
+    ratingName,
+    onRatingChange,
+    rating,
+  }: {
+    className?: string
+    ratingName: string
+    onRatingChange: (number: number) => void
+    rating: number
+  }) => {
+    const capitalizeFirstLetter = (str: string): string => {
+      return str.charAt(0).toUpperCase() + str.slice(1)
+    }
+
+    return (
+      <div className={className}>
+        <RatingLabel>{capitalizeFirstLetter(ratingName)}</RatingLabel>
+        <ReactStars
+          count={5}
+          char={'●'}
+          value={rating}
+          size={24}
+          color1={'#DDDDDD'}
+          color2={colours.primary2}
+          edit={true}
+          onChange={onRatingChange}
+        />
+      </div>
+    )
+  }
+)`
+  display: flex;
+  justify-content: space-between;
+  margin-left: 10px;
+  margin-right: 10px;
+  align-items: center;
+`
+
+const ModalCommentDiv = styled.div`
+  text-align: left;
+  margin: 10px;
+`
+
+const ModalComment = styled.textarea`
+  margin-top: 10px;
+  width: 100%;
+  height: 100px;
+  resize: none;
+  border: none;
+  background: #efefef;
+  border-radius: 5px;
+  outline: none;
+  padding: 7px;
+  font-family: ${fontInter};
+  font-size: 1em;
+`
+
+const AboutYou = styled.h3`
+  text-align: left;
+  width: 100%;
+  margin-left: 10px;
+  margin-right: 10px;
+`
+
+const CancelButton = styled.button`
+  background: white;
+  border: medium solid #ff6961;
+  border-radius: 5px;
+  font-family: ${fontInter};
+  font-weight: ${fontWeight.bold};
+  color: #ff6961;
+  width: 100px;
+  padding: 5px 20px;
```

Comment:
@CarolinaLi and @lamvpham originally designed whitespace in multiples of `8px` (`4px`, `8px, `16px`, etc etc). Could you adjust spacing  hroughout this entire PR?